### PR TITLE
adding client table and relationship with orders

### PR DESCRIPTION
**Pull request by eapg**
* Adding client table and relationship with orders

### CODE REVIEW

File: src/migrations/versions/d066d19614f3_adding_client.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+from lib.models.models import User, Client, Order
+
+# revision identifiers, used by Alembic.
+revision = 'd066d19614f3'
+down_revision = '31387db2a03a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```
Comment (reynoldmorel) [inline src/migrations/versions/d066d19614f3_adding_client.py:21]: Could you just insert it manually? like using a normal `Insert` statement?

Comment (reynoldmorel) [reply src/migrations/versions/d066d19614f3_adding_client.py:21]: You can also delete the models file for now.

We can define that later

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: src/migrations/versions/d066d19614f3_adding_client.py (hunk 1 above)
Comment (eapg) [reply src/migrations/versions/d066d19614f3_adding_client.py:20]: done

File: src/migrations/versions/31387db2a03a_client_table.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        sa.Column('user_id', sa.Integer(), nullable=False),
+        sa.Column('entity_status', postgresql.ENUM("ACTIVE", "DELETE", name="status_enum", create_type=False), nullable=False),
+        sa.Column('create_by', sa.Integer(), nullable=False),
+        sa.Column('create_date', sa.DateTime),
+        sa.Column('update_by', sa.Integer(), nullable=False),
+        sa.Column('update_date', sa.DateTime),
+        sa.PrimaryKeyConstraint('id')
+    )
+
+    op.add_column('orders', sa.Column('client_id', sa.Integer()))
```
Comment (reynoldmorel) [inline src/migrations/versions/31387db2a03a_client_table.py:33]: Client id should be not null

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: src/migrations/versions/31387db2a03a_client_table.py (hunk 2 above)
Comment (eapg) [reply src/migrations/versions/31387db2a03a_client_table.py:33]: done.

**Review by reynoldmorel — APPROVED** [review]

**Pull request merged**